docs/components/HeaderLogo.tsx

```diff
@@ -32,5 +32,3 @@ function HeaderLogo() {
     </>
   );
 }
-
-export default HeaderLogo;
```

Comment:
Very stylistic so I shouldn't care, but I like this change. I always miss it when there all the exports are at the bottom (some people really like this style and that's totally fine too)

Reply:
![CleanShot 2023-12-20 at 11 25 28@2x](https://github.com/vercel/turbo/assets/35677084/2f0593ed-c1ce-4819-afc4-39b05a603213)